src/content/en/updates/2017/02/improved-add-to-home-screen.md

```diff
@@ -61,9 +61,9 @@ and will be rolling out to Chrome 57 beta over the next few weeks.
     1. In the developer preview you'll additionally see a prompt from the 
        package installer to confirm.
 
-This new experience is a huge improvement over the original version of Add to Home
-screen, but there are some differences between these installed Progressive Web
-Apps and Android Apps.
+This new experience is a huge improvement over the original version of Add to
+Home screen, but there are some differences between these installed Progressive
+Web Apps and Android Apps.
 
```

Comment:
Hot button phrases, like 'Progressive Web Apps' are generally not capitalized either no matter how in vogue or how much we like them.